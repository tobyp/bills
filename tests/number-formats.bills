#!./bills
A	B.	1.23
C	D.	4,56
E	F.	7
