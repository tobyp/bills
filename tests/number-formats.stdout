todo: B pays 1.23 to A
todo: D pays 4.56 to C
todo: F pays 7.0 to E
